README.md

```diff
@@ -19,3 +19,18 @@ Clone this repo manually or use `brunch new dir -s brunch/with-react`
     * `public/` dir is fully auto-generated and served by HTTP server.  Write your code in `app/` dir.
     * Place static files you want to be copied from `app/assets/` to `public/`.
     * [Brunch site](http://brunch.io), [Getting started guide](https://github.com/brunch/brunch-guide#readme)
+
+## How to contribute
+Contribution is very welcome, you can review [current issues](https://github.com/algoristas/dashboard/issues) or request a new one by creating a new issue. 
+
+### Pull Request process
+Once you have the issue/feature to work on, follow these steps to submit your changes and  make a pull request of them.
+
+* Inside repository create a new branch `git checkout -b <your_branch>`
+* Made your changes and add them `git add <list of your files>`
+* Create a commit with an appropiate message `git commit` [guidelines](https://chris.beams.io/posts/git-commit/)
```

Comment:
I think most people know they have to commit their changes, I would just say something like: _Try to follow the guidelines at <the link> for your commits_.